cmd/lambda/update_lease_status/main.go

```diff
@@ -54,6 +56,12 @@ func main() {
 			log.Fatalf("Failed to configure Usage service %s", err)
 		}
 
+		// Configure the S3 service
```

Comment:
Of course, it would be nice to use the new configuration service and all that,
but I I didn't want to rabbit-hole on it.